frontend/src/utils/createKeycloakInstance.ts

```diff
@@ -0,0 +1,20 @@
+import IKeycloakConfig from 'interfaces/IKeycloakConfig';
+import Keycloak from 'keycloak-js';
+
+/**
+ * Fetches the configuration file `/keycloak.json` and creates a KeycloakInstance for it.
+ * Overrides file configuration with environment variables.
+ * @returns Promise<KeycloakInstance>
+ */
+export const createKeycloakInstance = async () => {
+  const response = await fetch('/keycloak.json');
+  const data = await response.json();
+  const config = data as IKeycloakConfig;
+
+  //@ts-ignore
+  return new Keycloak({
+    url: process.env.REACT_APP_KEYCLOAK_AUTH_SERVER_URL ?? config.authServerUrl,
```

Comment:
We can now override Keycloak configuration values locally with environment variables.

Reply:
Why is this desirable? this seems unrelated to the bugfix.

Reply:
It's unrelated to the bug, but required to support different port configurations.  Which is required to run two different versions of PIMS.